flake.nix

```diff
@@ -84,7 +84,10 @@
                 stdenv = pkgs.stdenvAdapters.useMoldLinker pkgs.clangStdenv;
               }
               {
-                packages = [ rust-toolchain ] ++ flakeshot.nativeBuildInputs ++ flakeshot.buildInputs;
+                packages = with pkgs; [
+                  snixembed
```

Comment:
Won't this clash if some other tray is installed?

Reply:
`snixembed` has to be started in the cli. So as long you don't start it, nothing happens.